package.json

```diff
@@ -65,7 +66,9 @@
     "cli-table": "^0.3.1",
     "commander": "^3.0.1",
     "dotenv": "^8.1.0",
+    "es6-promise": "^4.2.8",
```

Comment:
Remove this (`yarn remove es6-promise`). We shouldn't need this as all nodejs versions come with promise support

Reply:
Good catch, removed it!